docs/statcast_fielding.md

```diff
@@ -0,0 +1,123 @@
+`min_opp:` The minimum number of opportunities for the player to be included in the result. The default is players with at least 1 fielding attempt per game. 
+
+## Examples of Valid Queries
+```python
+from pybaseball import statcast_outfield_catch_proba
+
+# All qualified outfielders from 2019
+data = statcast_outfield_catch_proba(2019)
+
+# All outfielders with at least 200 attempts in 2019
+data = statcast_outfield_catch_proba(2019, 200)
+```
+
+# Statcast Fielding Outfielder Jump
+`statcast_outfielder_jump(year: int, min_att: Union[int, str] = "q")`
+
+This function retrieves data on outfielder's jump to the ball for the given year and number of attempts. Jump is calculated only for two star or harder plays (90% or less catch probabiility).
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`min_att:` The minimum number of attempts for the player to be included in the result. The default is players with at least 2 two star or harder fielding attempts per team game / 5. 
+
+## Examples of Valid Queries
+```python
+from pybaseball import statcast_outfielder_jump
+
+# All qualified outfielders from 2019
+data = statcast_outfielder_jump(2019)
+
+# All outfielders with at least 100 attempts in 2019
+data = statcast_outfielder_jump(2019, 100)
+```
+
+# Statcast Fielding Catcher Poptime
+`statcast_catcher_poptime(year: int, min_2b_att: int, min_3b_att: int)`
+
+This function retrieves pop time data for catchers given year and minimum stolen base attempts for second and third base. Pop time is measured as the time from the moment the ball hits the catcher's mitt to when it reaches the projected receiving point at the center of the fielder's base.
+
+Note: It is not available for 2020 data.
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`min_2b_att:` The minimum number of stolen base attempts for second base against the catcher.
+`min_3b_att:` The minimum number of stolen base attempts for third base against the catcher.
+
+## Examples of Valid Queries
+```python
+from pybaseball import statcast_catcher_poptime
+
+# Catchers with at least 10 second base attempts against and 2 third base attempts against in 2019
+data = statcast_catcher_poptime(2019, min_2b_att = 10, min_3b_att = 2)
+```
+
+# Statcast Fielding Catcher Framing
+`statcast_catcher_framing(year: int, min_called_p: Union[int, str] = "q")`
+
+This function retrieves the catcher's framing results for the given year and minimum called pitches. It uses eight zones around the strike zone (aka "shadow zone") and gives the percentage of time the catcher gets the strike called in each zone.
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`min_called_p:` The minimum number of called pitches for the catcher in the shadow zone. The default is players with at least 6 called pitches in the shadow zone per team game.
```

Comment:
Given how this recurs, maybe just say "Statcast's default" instead of "The default"